api/setup-minter.sh

```diff
@@ -2,35 +2,35 @@
 
 # Setup minter account
 
-curl --request POST \
-  --url http://localhost:3000/v1/kibbles/setup \
-  --header 'Content-Type: application/json'
+# curl --request POST \
+#   --url http://localhost:3000/v1/kibbles/setup \
+#   --header 'Content-Type: application/json'
 
-curl --request POST \
-  --url http://localhost:3000/v1/kitty-items/setup \
-  --header 'Content-Type: application/json'
+# curl --request POST \
+#   --url http://localhost:3000/v1/kitty-items/setup \
+#   --header 'Content-Type: application/json'
 
-curl --request POST \
-  --url http://localhost:3000/v1/market/setup \
-  --header 'Content-Type: application/json'
+# curl --request POST \
+#   --url http://localhost:3000/v1/market/setup \
+#   --header 'Content-Type: application/json'
 
 # Mint Kibble and Kitty Items
 
-curl --request POST \
-  --url http://localhost:3000/v1/kibbles/mint \
-  --header 'Content-Type: application/json' \
-  --data '{
-    "recipient": "'$FLOW_ADDRESS'",
-    "amount": 50.0
-  }'
-
-curl --request POST \
-  --url http://localhost:3000/v1/kitty-items/mint \
-  --header 'Content-Type: application/json' \
-  --data '{
-    "recipient": "'$FLOW_ADDRESS'",
-    "typeId": 1
-  }'
+# curl --request POST \
+#   --url http://localhost:3000/v1/kibbles/mint \
+#   --header 'Content-Type: application/json' \
+#   --data '{
+#     "recipient": "'${MINTER_ADDRESS}'",
+#     "amount": 50.0
+#   }'
+
+# curl --request POST \
+#   --url http://localhost:3000/v1/kitty-items/mint \
+#   --header 'Content-Type: application/json' \
+#   --data '{
+#     "recipient": "'${MINTER_ADDRESS}'",
+#     "typeId": 1
+#   }'
 
```

Comment:
Can you revert back to the old one? It should work as long as the `FLOW_ADDRESS` env var has been set